spark/Makefile

```diff
@@ -7,7 +7,7 @@ spark: build-spark push-spark clean-spark
 # Hadoop libs at that point (we grab the 2.6.1 libs, which are
 # appropriate for 1.5.2-with-2.6). Note that you'll need to re-test
 # Zeppelin (and it may not have caught up to newest Spark).
-TAG = 2.3.1_v2
+TAG = 2.3.1_v7
```

Comment:
@akuksin To update this image, increment Tag and run make spark, it will build and push image to repo